config.toml

```diff
@@ -57,6 +57,12 @@ logLevel = "TRACE"
 #
 checkForUpdates = true
 
+# Bot mode
+#
+# Default: false
+#
+enableBotMode = false
+
```

Comment:
This should not be a global config but rather per network and stored in the database.

Reply:
This option is now in the `irc_network` table, configurable from the network edit panel.